apps/performance.cc

```diff
@@ -6,7 +6,9 @@
 #include <cassert>
 #include <cstdlib>
 #include <cstring>
+#if defined(__x86_64__)
 #include <immintrin.h>
+#endif
 #include <iomanip>
 #include <iostream>
 
```

Comment:
Can you `#ifdef` out the `memcpy_workaround` nonsense below too? That's only to workaround AMD (not even just x86) issues.